lib/iex/endpoints/ref_data.rb

```diff
@@ -0,0 +1,19 @@
+module IEX
+  module Endpoints
+    module RefData
+      def isin_mapping(isins, options = {})
+        response = post('ref-data/isin', { token: publishable_token, isin: isins }.merge(options))
+
+        if response.is_a?(Hash) # mapped option was set
```

Comment:
I really dislike these APIs that return two different things depending on what you pass into them. Why don't we make two methods: `isin` and `mapped_isin`?

Reply:
As you mentioned in another comment the client should not be too opinionated. Keeping it in mind, I think it's better to copy the behavior of IEX endpoint and have two different responses depending on options in the single method. 
Again, if split the endpoint to two methods, the end-user can call `isin` endpoint passing there  `mapped: true` option what causes an error while the response parsing. So it requires an end-user to know the client-specific method `mapped_isin`.